defrag/modules/bugs.py

```diff
@@ -0,0 +1,184 @@
+# Defrag - centralized API for the openSUSE Infrastructure
+# Copyright (C) 2021 openSUSE contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Any, Dict, List, Tuple
+from defrag import app, LOGGER
+from defrag.modules.helpers import QueryObject
+from defrag.modules.helpers.sync_utils import as_async
+from defrag.modules.helpers.exceptions import BugzillaException, ParsingException, NetworkException
+import bugzilla
+from bugzilla import BugzillaError
+from xmlrpc.client import Fault
+from fastapi import Response, status
+import aiohttp
+from bs4 import BeautifulSoup
+import re
+
+
+URL = "bugzilla.opensuse.org"
+bzapi = bugzilla.Bugzilla(URL)
+
+__MOD_NAME__ = "bugs"
+
+
+@app.get("/" + __MOD_NAME__ + "/")
+async def root() -> Dict[str, str]:
+    return {
+        "module": "Bugzilla",
+        "description": "Get information about bugs on bugzilla.opensuse.org"}
+
+
```

Comment:
The suggestion to clarify your implementation was simply to go for something like this:
```python
@dataclass
class BugZillaEntry:
    id: int
    product: Optional[str]
    # etc, etc.
```
Since the data is immutable anyway you can even use a plain `Namedtuple` instead of a dataclass. Or you can sublcass `Namedtuple` if you feel like you need to override or add behaviour.

Reply:
Done.